### DISCUSSION TITLE
Package variant query

### DISCUSSION THREAD

**Discussion by SamarSidharth** [kptdev/kpt#4121] [General]
Is it compulsory to for the upstream and downstream repo to be registered in same namespace as package variant.

Follow up question is it a good practice to register a single repo multiple times in different namespace (eg a common upstream repo that contains blueprints with multiple downstream repos)

**Comment by johnbelamaric** [comment]
Suggest you move this discussion to [Nephio](https://github.com/nephio-project/nephio/discussions), where further work on Porch is being done.

But the answer is, yes, because permissions are managed by namespace, at least for now we don't try to do cross-namespace access requests. Those are complicated.

You can register the same repo multiple times, yes, that should work. But it's not super scalable right now. I want to see that addressed as part of https://github.com/nephio-project/nephio/discussions/450.

**Reply by SamarSidharth** [reply]
Thanks @johnbelamaric I have moved this discussion to Nephio https://github.com/nephio-project/nephio/discussions/507.
